adoc/admin-security-rbac.adoc

```diff
@@ -92,8 +92,13 @@ kubectl create rolebinding admin --clusterrole=admin --user=<USER-1> --user=<USE
 Administrators can update the authentication connector settings after {productname}
 deployment as follows:
 
-. Open the `dex` configmap in `<cluster name>/addons/dex/dex.yaml` .
-. Adapt ConfigMap by adding LDAP configuration to the connector section of the `config.yaml` data.
+. Run the following `kubectl` command to access to access Dex ConfigMap:
```

Comment:
Double "to access". 

Should be reworded to "... command to access the Dex ConfigMap:"